application/src/test/java/com/sanction/thunder/resources/VerificationResourceTest.java

```diff
@@ -203,11 +207,7 @@ public void testVerifyEmailWithHtmlResponse() {
   /* HTML Success Tests */
   @Test
   public void testGetSuccessHtml() {
-    String expected = "<div class=\"alert alert-success\">\n"
-        + "<center><strong>Success!</strong></br>Your account has been verified.</center>\n"
-        + "</div>\n"
-        + "<link rel=\"stylesheet\""
-        + " href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css\" />";
+    String expected = "fixtures/success-fixture.html";
```

Comment:
This seems wrong, shouldn't the result of `getSuccessHtml()` from `VerificationResource` be the actual HTML? Otherwise the user is going to get a string that says the filename instead of the actual HTML.